Library/Homebrew/extend/os/linux/unpack_strategy/tar.rb

```diff
@@ -0,0 +1,12 @@
+module UnpackStrategy
+  module GnuTar
+    def extract_to_dir(unpack_dir, basename:, verbose:, extra_args: [])
+      super(unpack_dir, basename: basename, verbose: verbose,
+            extra_args: ["--owner==0", "--group=0", "--no-same-owner"])
```

Comment:
```suggestion
            extra_args: ["--no-same-owner"] + extra_args)
```
`--owner=0 --group=0` are only meaningful when creating a tar archive.
See https://www.gnu.org/software/tar/manual/html_section/tar_33.html

The value of the argument `extra_args` should be passed on to `super`.

Reply:
Yes, only -o is needed

Reply:
I understand changes but last your comment/conversation cancels need in this file at all

Reply:
unused file